tests/pkcli/fmt_data/fmt_dir.in/y.py

```diff
@@ -0,0 +1,20 @@
+def force_arg_per_line_and_str(blah,this, be, a, very, poorly, formatted, function, that, noone, should, ever, use, it, has, way, too, many, arguments):
+    '''Messy Docstring
+If the element does not exist or is not indexable, fails silently with None.
+Args:
+dotted_key (str): what
+Returns:
+object: value of element or None
+        '''
+    x='Dw0kHyZ4DDIPMaLxZkRl2pFskoSzcDjSIU8czQA7H1PDXwhjQzDFY2lWeHmb1bKHSLEnlLf4bhTd'
+
+
+
+
+
+    y= 100
+    z= 200
+    p, q = 1, 2
+
+
+    print(y+z+p+q)
```

Comment:
why the print?

Reply:
@robnagler The print is random, for the code to document the test case should I call `foo(y+z+p+q)` instead to show that it is just placeholder?

Reply:
Yes, prints are only useful if you want to test the output. Rather just do something that doesn't cause output.